goalc/compiler/IR.cpp

```diff
@@ -1278,23 +1299,36 @@ IR_VFMath3Asm::IR_VFMath3Asm(bool use_color,
                              const RegVal* dst,
                              const RegVal* src1,
                              const RegVal* src2,
+                             u8 mask,
```

Comment:
I think we can remove `mask` from this IR entirely?  This IR always operates on all four components and doesn't do anything with the mask, other than print it.